locust/test/test_load_locustfile.py

```diff
@@ -209,3 +211,18 @@ def test_locustfile_can_be_set_in_config_file(self):
                 ]
             )
             self.assertEqual("my_locust_file.py", options.locustfile)
+
+    def test_locustfile_from_url(self):
+        locustfiles = parse_locustfile_option(
```

Comment:
Why not call parse_options instead? (as that tests more). Not a big issue though...